content/geodesic/module/_index.md

```diff
@@ -5,3 +5,49 @@ description: "Geodesic modules are Git repositories that extend the Geodesic bas
 {{% dialog type="info" icon="fa fa-book" title="Read More" %}}
 ⇠ Select one of these pages to learn more about the topic.
 {{% /dialog %}}
+
+We believe that infrastructure as code should be treated just like all other code: tested, bundled into artifacts and automatically deployed. This is what `geodesic` solves. A geodesic module is a docker container/image contains all the code necessary deploy infrastructure.
```

Comment:
A geodesic module is a docker container/image containing

Reply:
Grammatically, I think the sentence should be "A geodesic module is a docker container/image which contains all the code necessary to deploy infrastructure" except I still don't know what that means. Maybe it's supposed to be "...all the code necessary to deploy itself"?

Reply:
@Nuru thanks for checking this out.

How is this? I really want to make sure this is clear. If you don't understand it, others won't either.

> A geodesic module is a docker container/image which contains all the Infrastructure as Code necessary to provsion logically related IaaS resources (e.g. AWS Organizations, IAM Roles/Permissions, CDNs, S3 Buckets, Kubernetes Clusters, etc).